include/crm/crm.h

```diff
@@ -66,7 +66,7 @@ extern "C" {
  * >=3.0.13: Fail counts include operation name and interval
  * >=3.2.0:  DC supports PCMK_EXEC_INVALID and PCMK_EXEC_NOT_CONNECTED
  */
-#  define CRM_FEATURE_SET		"3.17.2"
+#  define CRM_FEATURE_SET		"3.17.3"
```

Comment:
Personally, I think it's enough to only bump CRM_FEATURE_SET once for this entire PR since everything's going in at the same time.  @kgaillot may want it to be more fine grained than that, though.